ingestion/src/metadata/ingestion/source/hive.py

```diff
@@ -18,24 +18,46 @@
 
 complex_data_types = ["struct", "map", "array", "union"]
 
+_type_map.update(
```

Comment:
nice

Reply:
I saw some imports at the bottom of the file. Could we pls move them to the top? Unrelated, but the most we can clean the better :P